### PR TITLE
Introduce connection specific exists query

### PR DESCRIPTION
**Pull request by georgehristov**

### CODE REVIEW

File: tests/OracleTest.php [hunk 1]
```diff
@@ -114,4 +114,12 @@ public function test12OracleLimitSkip()
             $c->dsql()->table('foo')->where('bar', 1)->field('baz')->limit(10, 99)->render()
         );
     }
+
+    public function testOracleExists()
```
Comment (mvorisek) [inline tests/OracleTest.php:118]: `OracleTest.php` file/tests will be dropped soon Oracle DB will be fully tested in CI, just FYI, we can merge now...

**Review by mvorisek — APPROVED** [review]
very good

File: tests/OracleTest.php (hunk 1 above)
Comment (georgehristov) [reply tests/OracleTest.php:118]: Then in testExists we should check for each connection? It is already complaining that Mssql\Query::exists is not tested...

Comment (mvorisek) [reply tests/OracleTest.php:118]: yes (and I expect that it is already tested indirectly in atk4/data)

**Pull request merged**